examples/evaluate_model.py

```diff
@@ -160,7 +160,10 @@ def export_model_preds(preds_file, predictions, validation_stream, verbalizer):
         )
     )
     export_model_preds(
-        args.preds_file, predictions, validation_stream, model.verbalizer
+        args.preds_file,
+        predictions,
+        validation_stream,
+        getattr(model, "verbalizer", None),
```

Comment:
QQ: model will not contain `verbalizer` in case of  non-prompt tuning ones, right? (Just checking)